2200059/5431_민석과제체크.py

```diff
@@ -0,0 +1,13 @@
+# import sys
+# sys.stdin = open("input.txt")
+T = int(input())
+for t in range(1, T + 1):
+    N, K = map(int, input().split())
+    students = list(map(int, input().split()))
+    print(f'#{t}', end=' ')
+    for i in range(1, N+1):
+        if i in students:
+            pass
+        else:
+            print(i, end=' ')
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    for i in range(1, N+1):
        if i not in students:
            print(i, end=' ')
```
위와 같이 작성하셔도 좋을 것 같아요 😄